TakingITGlobal/commit2act
# [TIG-748, TIG-742] Updates from Signup flow designs and alternate back/next links

## Ticket(s)

* [TIG-748](https://app.clickup.com/t/9009201449/TIG-748)
* [TIG-742](https://app.clickup.com/t/9009201449/TIG-742)

## Description

This PR is a bit of a mixed bag, but comes following design revisions prompted by developer notes. These updates are generally intended to encourage users to enter their primary info up front before offering the option to skip the rest of the onboarding flow. Button labels, page order, percentage, etc. are all geared towards this. 

The main elements of this PR include:

* Moving `InfoSecondary` to step 2 and `OnboardStart` to step 3
* Introducing alternate labels and paths for next/back links throughout pages
* Adding the "Skip for now" link on the `OnboardStart` page
* Updating progress values to reflect the percentage completed
* Update heading from temp placeholder on final `OnboardEnd` page

## Testing

1. Pull down and install the branch (`npm install`)
2. Run all tests (`npm run test`) and confirm they pass
3. Run Storybook locally (`npm run storybook`) and visit the Pages > Sign Up section
4. Confirm the first step shows 0% for progress and others still follow consistent increments
5. Confirm `InfoSecondary` is now Step 2 and `OnboardStart` is Step 3
6. Confirm `OnboardStart` has the "Skip for now" link
7. Confirm each step page has back/next links matching designs
  a. `InfoPrimary` now uses "Continue", as `InfoPrimary` assumes the "Complete" button label
  b. `OnboardStart` and `OnboardEnd` should both only have one button 
8. Confirm `OnboardEnd` features the new designed heading (with 3 exclamation points)
9. Run the app locally (`npm start`) and visit the first step at `/account/create`
10. Use the "Continue" link and confirm step 2 renders at `/account/create/details`
11. Navigate through each step and confirm the links work as expected

## Screenshots

**First step with 0% progress and "Continue" button text:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/4771fe9d-356f-409c-8e5b-a1200dbfe638)

**Third step is now `OnboardStep` with a single "Let's Go" main action and the "Skip for now" secondary link:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/19f7f83b-9374-4ca8-8581-fa1df26412df)

**Final step with single "Start Acting" button:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/504e2169-36c5-41bc-94f6-657e744f7a80)

**Storybook entry with steps 2 and 3 swapped:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/f756ec54-a08c-48d8-bbde-ec475cd45bfd)
